.gitignore

```diff
@@ -2,6 +2,9 @@ node_modules
 storybook-static
 .idea
 
+# yalc stuff
+.yalc
+yalc.lock
```

Comment:
This can be removed, can't it?

Reply:
To clarify, I don't think this is broadly applicable to every person who clones this repo.  If you use it personally, and want to gitignore it, you can add it to a personal ignore file or `.git/info/exclude` (https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file)

Reply:
Yes thanks for the clarification, happy to remove.